Favor/Favor/Sources/Scenes/AnniversaryList/Reactors/AnniversaryListViewReactor.swift

```diff
@@ -32,13 +32,15 @@ final class AnniversaryListViewReactor: BaseAnniversaryListViewReactor, Reactor,
   enum Mutation {
     case updateAnniversaries([Anniversary])
     case updateAllSection([Item])
+    case updateToast(Bool)
   }
   
   struct State {
     var anniversaries: [Anniversary] = []
     var sections: [Section] = []
     var items: [Item] = []
     var anniversaryListType: AnniversaryListType
+    var shouldShowToast: Bool = false
```

Comment:
```suggestion
    @Pulse var shouldShowToast: ToastMessage?
```

`ReactorKit`의 `@Pulse` 프로퍼티 래퍼를 활용하면 좋을 것 같아!

``` Swift
.just(.updateToast(true)),
.just(.updateToast(false))
```

그러면 이렇게 같은 프로퍼티에 대한 값을 바로 껐다 켜는 대신

``` Swift
.just(.presentToast(.maxPinExceeded))
```

이렇게 깔끔하게 사용할 수 있어..!